### ISSUE TITLE
GeluidschermdeelMetDiffractor in geval van industrie

### ISSUE THREAD

**Issue by PB-GNM** [Geonovum/IMG#223]
# Aanleiding wijziging
Dit verzoek is ingediend door DGMR met helpdesknummer SDIMG-146 en met de volgende motivatie:
Volgens het schema kan voor industrie een  GeluidschermdeelMetDiffractor worden gedefinieerd.
Het rekenvoorschrift voor industrie ondersteunt geen diffractoren, dus ook geen geluidschermen met diffractor er bovenop. Het is niet duidelijk op welke wijze deze items voor industrie moeten worden berekend.
Dit is relevant omdat GPP-spoor ook industriële bronnen bevat (rangeerterreinen) in combinatie met diffractoren op schermen

# Voorgestelde wijziging
Toelichting opnemen bij GeluidschermdeelMetDiffractor hoe in de berekeningen om te gaan in het geval van industrie:
_"Een GeluidsschermdeelMetdiffractor is alleen van toepassing op geluid van wegen en spoorwegen."_.
De geluidschermdeelMetDiffractor zal dan ook uit [het diagram voor GPP-Industrie](https://docs.geostandaarden.nl/cvgg/img/#EAID_4D140CCE_0889_4998_B668_8E4DB0053589) gehaald worden.

![Image](https://github.com/user-attachments/assets/80907dba-70e4-4990-b6f6-b864b79bc881)

# Impactanalyse
- Wie gaat er wat van merken? **softwareleveranciers**
- Veranderen definities van objecten in de standaard zodanig dat de wijziging impact heeft op de uit te wisselen gegevens? **Nee**
- Is er een herlevering nodig naar de CVGG? **Nee**
- Heeft het impact op het xml-schema? **Nee**
- Is het backward compatible in de zin van validatie op het schema? **Ja**
- Heeft het impact op de regels in IMG? **Nee**
- Heeft het impact op de validatieregels voor de CVGG? **Nee**
- Heeft het impact op de omgevingsregeling? **Nee**
- Is het een X, Y, of Z wijziging? **Z**

# Toelichting
Ruimte voor extra toelichting. Bijvoorbeeld alternatieven die je overwogen hebt.

**Comment by PB-GNM** [comment]
Reactie DGMR:
“GeluidschermdeelMetDiffractor” kunnen zowel in wegverkeer als spoorwegverkeer voorkomen. Is het gemeten diffractor effect van zo’n item langs een baanvak gelijk aan het diffractor effect als deze langs een weg staat en omgekeerd?

**Comment by ArnaudYK** [comment]
Ja. Het diffractoreffect is een eigenschap van de diffractor. Het is onafhankelijk van de bron.

**Comment by ArnaudYK** [comment]
Met betrekking tot de toelichting moet je nadenken wat je wilt. Je geeft nu een modelleerrichtlijn (wat op zich wel klopt). Het is denk ik wel een keuze of dat wel een plek heeft in het IMG. Je zou ook kunnen schrijven dat het effect van een diffractor alleen in rekening gebracht kan worden bij weg en railverkeer (Bijlage IVe en IVf van de OR).

**Comment by PB-GNM** [comment]
Tijdens de adviesgroep van 24 oktober is dit issue niet besproken. Het zal in een volgend adviesgroepgesprek besproken worden.

**Comment by PB-GNM** [comment]
Op verzoek van RIVM is het deel "_In overige gevallen wordt het beschouwd als geluidsschermdeel met profieltype scherp_" dat als toelichting in het voorstel stond uit de voorgestelde toelichting gehaald.

**Comment by wilkoquak** [comment]
Uitkomst adviesgroep 30 januari:
- De toelichtende tekst wordt niet toegevoegd. Omdat dit gaat over de interpretatie van het reken- en meetvoorschrift vindt de adviesgroep het niet handig deze toelichting in het IMG op te nemen.
- De klasse GeluidschermdeelMetDiffractor wordt wel verwijderd uit het diagram voor GPP-industrie.
